lib/assessment.js

```diff
@@ -38,9 +65,19 @@ var newArray2 = function newArray2(length, defaultsFunction) {
 // If the argument obj is unset, create a new object
 // The function you write should return
 // the object passed in as an argument or the object created
-var addMethod = function addMethod(propertyName, method, obj) {
+
+var addMethod = function addMethod(propertyName, method, obj) //if it adds or replaces a method, method has to be an argument, and if there's an unset object, object has to be a argument too
+ {
+  if(obj === undefined) {
+    obj = {};
+  }
+
+  obj.propertyName = method;
```

Comment:
This will always assign the method to the object's property called "propertyName", not the string that was passed in. You must use square brackets in order for the variable to accessed and the string within used as the property name.